Update with MD API reasonable use info (#3295)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud-apis/metadata-querying.md

There are several illustrative example queries in this documentation. You can see an example of [queries on the Model node](/docs/dbt-cloud-apis/metadata-schema-model).

## Reasonable use

> All Metadata (GraphQL) API requests are subject to request rate and response size limiting to prevent abuse and help ensure performance and stability. 

Reviewer: ```suggestion
To maintain performance stability and prevent abuse, all Metadata (GraphQL) API requests are subject to request rate and response size limits.
```

> The current rate limit is 200 requests within a minute for a given IP address. If a user exceeds this limit, they will receive a HTTP 429 response status.

Reviewer 2: What does response size limiting look like for us? You mention it, but unlike the request rate limit there's no context on how that might manifest or how to be a responsible API consumer. 

For an example, GH's GraphQL API limits the number of nodes queryable to 500k: https://docs.github.com/en/graphql/overview/resource-limitations#node-limit

Reviewer: just curious and this may be an obvious question - is there a warning for users if they get close to this limit?  or a way for the user to check ?

Author: @joellabes great point, we added response size limit details. we haven't implemented this yet, but plan to soon. the node limit will likely change, but want to establish expectations for usage best practices

@mirnawong1 good idea, but at least for now upon hitting the limit, the user will just receive the error and need to retry. we calibrated the rate limit to the far upper end of API usage

> The current rate limit is 200 requests within a minute for a given IP address. If a user exceeds this limit, they will receive a HTTP 429 response status.

Reviewer: ```suggestion
The current rate limit is 200 requests within a minute for a given IP address. If exceed this limit, you'll receive an HTTP 429 response status.
```

5. Run your query by pressing the blue query button in the top-right of the Operation editor (to the right of the query). You should see a successful query response on the right side of the explorer.

> <Lightbox src="/img/docs/dbt-cloud/metadata-api/graphql.jpg" width="85%" title="Run queries using the Apollo Server GraphQL explorer"/>.

Reviewer 2: ```suggestion
<Lightbox src="/img/docs/dbt-cloud/metadata-api/graphql.jpg" width="85%" title="Run queries using the Apollo Server GraphQL explorer."/>
```
